www/js/controllers.js

```diff
@@ -25,6 +25,48 @@ module.run(function($ionicPlatform) {
 	});
 });
 
+var LoginAlerts = function () {};
+
+LoginAlerts.Helper_ErrorAlert = function ($ionicPopup, message) {
+	var alertPopup = $ionicPopup.alert(
+	{
+		title: '<span class="bold error">Error!</span>',
+		template: message
+	});
+};
+
+LoginAlerts.Helper_SuccessAlert = function ($ionicPopup, message) {
+	var alertPopup = $ionicPopup.alert(
+	{
+		title: '<span class="bold success">Success!</span>',
+		template: message
+	});
+};
+
+LoginAlerts.Error_AlreadyLoggedIn = function ($ionicPopup) {
```

Comment:
keep the generic ones in the LoginAlerts class and specific ones as helper methods in their controllers if they need to be reused.